src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Nullable.cs

```diff
@@ -1155,11 +1144,7 @@ static void G(object o)
                 source,
                 references: new[] { ref0 },
                 parseOptions: TestOptions.Regular8);
-            comp.VerifyEmitDiagnostics(
-                // error CS0518: Predefined type 'System.Attribute' is not defined or imported
-                Diagnostic(ErrorCode.ERR_PredefinedTypeNotFound).WithArguments("System.Attribute").WithLocation(1, 1),
-                // error CS0518: Predefined type 'System.Attribute' is not defined or imported
-                Diagnostic(ErrorCode.ERR_PredefinedTypeNotFound).WithArguments("System.Attribute").WithLocation(1, 1));
+            comp.VerifyEmitDiagnostics();
```

Comment:
>VerifyEmitDiagnostics [](start = 17, length = 21)

:memo: No module-level `Nullable` attribute is synthesized, so no dependency on missing `Attribute` type.